tests/publiccloud/azure_aitl.pm

```diff
@@ -0,0 +1,133 @@
+# SUSE's openQA tests
+#
+# Copyright 2021-2024 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Create VM in Azure using azure-cli binary
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use mmapi 'get_current_job_id';
+use utils qw(zypper_call script_retry);
+use version_utils 'is_sle';
+use registration qw(add_suseconnect_product get_addon_fullname);
+use JSON;
+use XML::LibXML;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    my $job_id = get_current_job_id();
+
+    # If 'az' is preinstalled, we test that version
+    if (script_run("which az") != 0) {
```

Comment:
we need to discuss this part , we should rely on the fact that tools image provide everything what you need

Reply:
I just copied from the `azure_cli` test module, so let's discuss.

Reply:
Removed this part as it was unnecessary.